cdap-examples/WikipediaPipeline/src/main/java/co/cask/cdap/examples/wikipedia/package-info.java

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
I can't tell on github if it's lined up or not...if it looks fine to you, it's ok